clang/test/Driver/netbsd.c

```diff
@@ -461,3 +461,8 @@
 // DRIVER-PASS-INCLUDES:      "-cc1" {{.*}}"-resource-dir" "[[RESOURCE:[^"]+]]"
 // DRIVER-PASS-INCLUDES-SAME: "-internal-isystem" "[[RESOURCE]]{{/|\\\\}}include"
 // DRIVER-PASS-INCLUDES-SAME: {{^}} "-internal-externc-isystem" "{{.*}}/usr/include"
+
+// Check that the driver passes the system library path.
```

Comment:
This file is very long and can be better organized. I think you can reuse an existing RUN line.

If cmake `DEFAULT_SYSROOT` is set, there may not be a  `-L/usr/lib`. You need to set `--sysroot=` to test a usr/lib directory